core/src/main/java/com/netease/arctic/op/CreateTableTransaction.java

```diff
@@ -38,6 +38,7 @@
 import org.apache.iceberg.UpdateProperties;
 import org.apache.iceberg.UpdateSchema;
 import org.apache.iceberg.relocated.com.google.common.collect.Lists;
+
```

Comment:
Do you format the hold project?

Reply:
Not yet, I formatted the imports in the `ams-server` module.

Reply:
I think you can format the files you modified. If you haven't made any changes, try to keep the original format, which is more friendly for reviews or future cherry-picks.

Reply:
We should strive to maintain uniform code formatting to avoid different developers using different code styles. 
Look forward to your PR for automatically formatting code based on Spotless.